contracts/test/branded_token/TestBrandedToken.sol

```diff
@@ -0,0 +1,119 @@
+pragma solidity ^0.4.24;
+
+// Copyright 2018 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @title TestBrandedToken.
+ *
+ * @notice It's needed for testing branded token contract.
+ */
+contract TestBrandedToken {
+
+    /* Events */
+
+    event StakeRequested(
+        bytes32 indexed _stakeRequestHash,
+        address _staker,
+        uint256 _stake,
+        uint256 _nonce
+    );
+
+    event StakeRequestAccepted(address _staker, uint256 _stake);
+
+    event Transfer(
+        address indexed _from,
+        address indexed _to,
+        uint256 _value
+    );
+
+
+    /* Special Functions */
+
+    /**
+     * @notice Contract constructor.
+     *
+     * @dev Ordering of constructor parameters:
+     *      valueToken
+     *      symbol
+     *      name
+     *      decimals
+     *      conversionRate
+     *      conversionRateDecimals
+     *      Organization contract address
+     */
+    constructor(
+        address,
+        string,
+        string,
+        uint8,
+        uint256,
+        uint8,
+        address
+    )
+        public
+    {}
+
+
+    /* External Functions */
+
+    /**
+     * @notice Mocks BT requestStake method.
+     *
+     * @dev It takes below parameters in order:
+     *      - Amount to Stake.
+     *      - Amount to Mint.
+     *
+     * @return unique hash for each stake request.
+     */
+    function requestStake(
+        uint256,
+        uint256
+    )
+        external
+        returns (bytes32)
+    {
+        emit StakeRequested(bytes32(0), address(0), uint256(0), uint256(0));
+
+        return bytes32(0);
+    }
+
+    /**
+     * @notice Mocks BT acceptStakeRequest method.
+     *
+     * @dev It takes below parameters in order:
```

Comment:
For each item in this list, start lowercase and please remove the period at the end.